ui/app/AppLayouts/Communities/panels/SharedAddressesPermissionsPanel.qml

```diff
@@ -143,6 +145,21 @@ Rectangle {
                     font.weight: Font.Medium
                     text: qsTr("Permissions")
                 }
+                Item { Layout.fillWidth: true }
+                RowLayout {
+                    Layout.rightMargin: Style.current.halfPadding
+                    spacing: 4
+                    visible: root.requirementsCheckPending
+                    StatusBaseText {
+                        text: qsTr("Updating eligibility")
+                        font.pixelSize: Style.current.tertiaryTextFontSize
+                        color: Theme.palette.baseColor1
+                    }
```

Comment:
Ideally texts should be always protected from overflows when used with translations which may be potentially longer.

![Screenshot from 2024-03-27 11-07-36](https://github.com/status-im/status-desktop/assets/20650004/4f135b04-1e7d-48db-b0a3-1b20c78b3423)